src/localisation/language.cpp

```diff
@@ -41,6 +41,7 @@ enum {
 	RCT2_LANGUAGE_ID_CHINESE_TRADITIONAL,
 	RCT2_LANGUAGE_ID_12,
 	RCT2_LANGUAGE_ID_PORTUGUESE,
+    RCT2_LANGUAGE_ID_NORWEGIAN,
```

Comment:
This is a mapping for the languages in the original game. You probably shouldn't modify this enum, and use English UK below instead.